src/providers/url/url.ts

```diff
@@ -31,4 +31,9 @@ export class UrlProvider {
     return urlTemplate.replace('{staffNumber}', isNil(staffNumber) ? '00000000' : staffNumber);
   }
 
+  getRekeyFindUserUrl(staffNumber: string): string {
+    const urlTemplate = this.appConfigProvider.getAppConfig().user.findUser;
```

Comment:
is there a corresponding PR to add this URL to the config service?

Reply:
Yup, thats merged already in dev, should I add the pr

Reply:
cool nw